### PR TITLE
create-seeder

### PR DESCRIPTION
**Pull request by giapnguyenitf**

### CODE REVIEW

**Comment by giapnguyenitf** [comment]
ready

File: database/factories/CategoryFactory.php [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?php
+
+use Faker\Generator as Faker;
+
+$factory->define(App\Models\Category::class, function (Faker $faker) {
+    return [
+        'name' => $faker->name,
+        'user_id' => function() {
+                return factory(App\Models\User::class)->create()->id;
```
Comment (trangkarol) [inline database/factories/CategoryFactory.php]: Indent: 4 space

File: database/factories/FavoriteFactory.php [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<?php
+
+use Faker\Generator as Faker;
+
+$factory->define(App\Models\Favorite::class, function (Faker $faker) {
+    return [
+        'user_id' => function()
+            {
```
Comment (trangkarol) [inline database/factories/FavoriteFactory.php]: Convention: 
> 'user_id' => function()  {

File: database/factories/FavoriteFactory.php [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+                return factory(App\Models\User::class)->create()->id;
+            },
+        'document_id' => function()
+            {
```
Comment (trangkarol) [inline database/factories/FavoriteFactory.php]: Convention: 
> 'document_id' => function()  {

File: database/factories/FriendshipFactory.php [hunk 4]
```diff
@@ -0,0 +1,17 @@
+<?php
+
+use Faker\Generator as Faker;
+
+$factory->define(App\Models\Friendship::class, function (Faker $faker) {
+    return [
+        'first_user' => function()
+            {
```
Comment (trangkarol) [inline database/factories/FriendshipFactory.php]: Convention: 
> 'first_user' => function()  {

File: database/factories/FriendshipFactory.php [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+                return factory(App\Models\User::class)->create()->id;
+            },
+        'second_user' => function()
+            {
```
Comment (trangkarol) [inline database/factories/FriendshipFactory.php]: Convention: 
> 'second_user' => function()  {

File: database/migrations/2018_01_02_080212_change_name_field_to_comments_table.php [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('comments', function (Blueprint $table) {
+            $table->renameColumn('messsages','messages');
```
Comment (trangkarol) [inline database/migrations/2018_01_02_080212_change_name_field_to_comments_table.php]: Convention: Sau dấu `,` là 1 space

File: database/migrations/2018_01_02_080212_change_name_field_to_comments_table.php [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('comments', function (Blueprint $table) {
+            $table->renameColumn('messages','messsages');
```
Comment (trangkarol) [inline database/migrations/2018_01_02_080212_change_name_field_to_comments_table.php]: Convention: Sau dấu `,` là 1 space

File: database/seeds/CategoriesTableSeeder.php [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+class CategoriesTableSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        factory(App\Models\Category::class,5)->create();
```
Comment (trangkarol) [inline database/seeds/CategoriesTableSeeder.php]: Convention: Sau dấu `,` là 1 space

File: database/seeds/CoinsTableSeeder.php [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+class CoinsTableSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        factory(App\Models\Coin::class,1)->create();
```
Comment (trangkarol) [inline database/seeds/CoinsTableSeeder.php]: Convention: Sau dấu `,` là 1 space

File: database/seeds/CommentsTableSeeder.php [hunk 10]
```diff
@@ -0,0 +1,16 @@
...
+class CommentsTableSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        factory(App\Models\Comment::class,5)->create();
```
Comment (trangkarol) [inline database/seeds/CommentsTableSeeder.php]: Convention: Sau dấu `,` là 1 space

File: database/seeds/DocumentsTableSeeder.php [hunk 11]
```diff
@@ -0,0 +1,16 @@
...
+class DocumentsTableSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        factory(App\Models\Document::class,5)->create();
```
Comment (trangkarol) [inline database/seeds/DocumentsTableSeeder.php]: Convention: Sau dấu `,` là 1 space

File: database/seeds/FavoritesTableSeeder.php [hunk 12]
```diff
@@ -0,0 +1,16 @@
...
+class FavoritesTableSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        factory(App\Models\Favorite::class,5)->create();
```
Comment (trangkarol) [inline database/seeds/FavoritesTableSeeder.php]: Convention: Sau dấu `,` là 1 space

File: database/seeds/TransactionsTableSeeder.php [hunk 13]
```diff
@@ -0,0 +1,16 @@
...
+class TransactionsTableSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        factory(App\Models\Transaction::class,5)->create();
```
Comment (trangkarol) [inline database/seeds/TransactionsTableSeeder.php]: Convention: Sau dấu `,` là 1 space

File: database/seeds/UsersTableSeeder.php [hunk 14]
```diff
@@ -0,0 +1,16 @@
...
+class UsersTableSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        factory(App\Models\User::class,5)->create();
```
Comment (trangkarol) [inline database/seeds/UsersTableSeeder.php]: Convention: Sau dấu `,` là 1 space

File: database/factories/CategoryFactory.php [hunk 15]
```diff
@@ -0,0 +1,12 @@
+<?php
+
+use Faker\Generator as Faker;
+
+$factory->define(App\Models\Category::class, function (Faker $faker) {
+    return [
+        'name' => $faker->name,
+        'user_id' => function() {
```
Comment (trangkarol) [inline database/factories/CategoryFactory.php]: Covention: `function ()`

**Comment by trangkarol** [comment]
commented

**Comment by giapnguyenitf** [comment]
ready

**Pull request merged**